modules/programs/vim.nix

```diff
@@ -39,6 +42,26 @@ in {
       '';
     };
 
+    programs.vim.customPlugins = mkOption {
+      type = types.attrsOf types.package;
+      default = {};
+      example = {
```

Comment:
Examples referring to derivations should use [`literalExample`](https://github.com/LnL7/nix-darwin/blob/master/modules/nix/nixpkgs.nix#L44-L53). This avoids evaluating the derivation when using darwin-option or when building a manual (if I add one).